### ISSUE TITLE
Song detection issues on rdio

### ISSUE THREAD

**Issue by NelsonTC** [david-sabata/web-scrobbler#449]
Hello.

I use Last.fm Scrobbler daily with rdio and I noticed a weird problem with the artist/song title detection issue.
For instance, just now I started playing my radio and second song was Devin Townsend's ZTO (1:17 long) and the extension showed the grey icon on the address bar. Right after, it played Meshuggah's Bleed (7:22 long) and I noticed the popup saying it scrobbled Devin Townsend - Bleed.

While playing the forth song on the list, my Last.fm account shows only one song with the 2nd artist name and track duration but the 3rd song title.

While writing this description I listened to Katatonia - The Parting and Mastodon - Oblivion, but got Katatonia - The Parting and Katatonia - Oblivion scrobbled.

Long story short, after 5 plays on rdio, my Last.fm listed 3 scrobbles, 2 of which had mixed artist/titles.

This is ruining my library and adding a lot of phony tracks to Last.fm. :(

Let me know if you need additional info.

Cheers, Nélson.

**Comment by inverse** [comment]
No longer a service
